integration_test.go

```diff
@@ -122,3 +122,45 @@ func TestSelectOne(t *testing.T) {
 		t.Errorf("Expect to find user with email == %s, got %s", "foo@bar.com", email)
 	}
 }
+
+func countUsers(t *testing.T, dot *DotSql, db QueryRower, name string, expected int) {
+	t.Helper()
+	row, err := dot.QueryRow(db, name)
+	if err != nil {
+		t.Error(err)
+	}
+	var count int
+	err = row.Scan(&count)
+	if err != nil {
+		t.Error(err)
+	}
+	if expected != count {
+		t.Errorf("expected %d users but got %d", expected, count)
+	}
+}
+
+func TestSelectData(t *testing.T) {
+	db, dotsql := initDotSql()
+	defer db.Close()
+
+	_, err := dotsql.Exec(db, "create-user", "foo", "foo@bar.com")
+	if err != nil {
+		panic(err)
```

Comment:
t.Error/Fatal?

Reply:
Yup. There were a few more of those lurking around in existing code, I've fixed them up too.